orchestra/todos/import_export.py

```diff
@@ -0,0 +1,122 @@
+import csv
+import json
+
+from django.conf import settings
+from django.db import transaction
+from django.utils import timezone
+from tempfile import NamedTemporaryFile
+
+from orchestra.core.errors import SpreadsheetImportError
+from orchestra.google_apps.permissions import write_with_link_permission
+from orchestra.google_apps.service import Service
+from orchestra.google_apps.convenience import get_google_spreadsheet_as_csv
+from orchestra.models import TodoListTemplateImportRecord
+
+
+REMOVE_IF_HEADER = 'Remove if'
+SKIP_IF_HEADER = 'Skip if'
+
+
+def _write_template_rows(writer, todo, depth):
```

Comment:
Took me a minute to get this, perhaps worth a docstring like "a row looks like `{remove_if_json}, {skip_if_json}, {empty_columns if needed so that children are nested underneath parents}, {todo_item_description}`"

Reply:
noic